app/com/arpnetworking/metrics/portal/reports/impl/chrome/DevToolsServiceWrapper.java

```diff
@@ -113,23 +140,26 @@ private void configureRequestInterception() {
         if (_closed.get()) {
             throw new IllegalStateException("cannot interact with closed devtools");
         }
-        return supplyInExecutor(() -> Base64.getDecoder().decode(_dts.getPage().printToPDF(
-                false,
-                false,
-                false,
-                1.0,
-                pageWidthInches,
-                pageHeightInches,
-                0.4,
-                0.4,
-                0.4,
-                0.4,
-                "",
-                true,
-                "",
-                "",
-                true
-        )));
+        return supplyInExecutor(() -> Base64.getDecoder().decode(
+                _dts.getPage().printToPDF(
+                    false,
```

Comment:
nit: It would be helpful to add a comment after each literal argument as to the parameter name to make this more readable.